app/screens/send-bitcoin-screen/scanning-qrcode-screen.tsx

```diff
@@ -72,6 +99,26 @@ export const ScanningQRCodeScreen = () => {
       return
     }
 
+    if (data.startsWith("200")) {
+      setPending(true)
+      try {
+        Alert.alert(`fetching billpay ${data}`)
+        const billpayEncoded = await getBillpayData(data)
+
+        if (billpayEncoded) {
+          Alert.alert(JSON.stringify(billpayEncoded))
+          // const d = { username: "nicolas", paymentType: "username" }
+          navigate("sendBitcoin", { payment: billpayEncoded })
+          return
```

Comment:
If billpayEncoded is false or errors the function won't return yet and run the validPayment function. Is this intended behavior?